### PR TITLE
fix/866_start_preview_page_number_at_1_in_api

### PR DESCRIPTION
**Pull request by inkhey**
related to tracim/tracim#866

### CODE REVIEW

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -797,6 +795,7 @@ def get_pdf_preview_path(
         :return: preview_path as string
         """
         file_path = self.get_one_revision_filepath(revision_id)
+        page = preview_manager_page_format(page)
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: `page` is probabbly `page_id` or`page_number`? Do not use page as it is intuitively an object (and here, my feeling is that it is a number)

File: backend/tracim_backend/lib/core/content.py [hunk 2]
```diff
@@ -847,6 +846,7 @@ def get_jpg_preview_path(
...
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: same remark

File: backend/tracim_backend/lib/core/content.py [hunk 3]
```diff
@@ -847,6 +846,7 @@ def get_jpg_preview_path(
...
         if page >= self.preview_manager.get_page_nb(file_path):
             raise Exception(
```
Comment (lebouquetin) [inline backend/tracim_backend/lib/core/content.py]: I believe raise a generic Exception is a wrong idea

Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py]: You're right, that's an error (specific exception for this already exist in the same file)

**Pull request merged**